dev/README.md

```diff
@@ -52,22 +50,13 @@ Check the [localconfig-example](bin/localconfig-example) file for an example of
 
 A remote Managed Kafka instance is required for the internal communication between components of the system.
 
-The [kafka-setup.sh](bin/kafka-setup.sh) script takes care of configuring it for you.
-It is **idempotent**, so it can be run whenever you want to make sure the Kafka cluster is configured properly.
+The development team is asked to use the *shared kafka instance* we have deployed under our organization. with the following with the service accounts you can download.
```

Comment:
could you please revisit this piece of documentation?

Reply:
Do you have any suggestion?

Reply:
I mean, could you please clarify _with the following with the service accounts you can download._

Reply:
Done